support/util/imagemetadata.go

```diff
@@ -183,7 +182,10 @@ func (r *RegistryClientImageMetadataProvider) GetDigest(ctx context.Context, ima
 	case len(composedParsedRef.Tag) > 0:
 		desc, err := repo.Tags(ctx).Get(ctx, composedParsedRef.Tag)
 		if err != nil {
-			return "", nil, err
+			fmt.Printf("failed to get repository tags for %s composedParsedRef: %+v: %v. Falling back to the original imageRef %s.\n", composedParsedRef.Tag, composedParsedRef, err, imageRef)
```

Comment:
Is this raw Printf a left over? This should be using the logger

Reply:
Agree on the logger, I will fix it on a follow up PR

Reply:
https://github.com/openshift/hypershift/pull/6204